.ci_support/exclude_fenics

```diff
@@ -0,0 +1,6 @@
+fenics_linear_elasticity_exercises.ipynb
```

Comment:
```suggestion
```

Reply:
this was already excluded in the initial `exclude` file, thus I didn't want to change that.

Reply:
oh ok I didn't know that